docs/breaking-changes.md

```diff
@@ -103,7 +103,22 @@ shell.trashItem(path).then(/* ... */)
 
 ## Planned Breaking API Changes (11.0)
 
-There are no breaking changes planned for 11.0.
+### Removed: `BrowserView.{fromId, fromWebContents, getAllViews}` and `id` property of `BrowserView`
+The experimental APIs `BrowserView.{fromId, fromWebContents, getAllViews}`
+have now been removed. Additionally, the `id` property of `BrowserView`
+has also been removed.
+
+For more detailed information, see [#23578](https://github.com/electron/electron/pull/23578).
+
+### Removed: `desktopCapturer.getMediaSourceIdForWebContents()`
```

Comment:
This API was both added and removed during the 11.x cycle, so its removal is not a breaking change.